spec/controllers/miq_report_controller/dashboards_spec.rb

```diff
@@ -102,9 +103,9 @@
         allow(controller).to receive(:db_fields_validation)
         allow(controller).to receive(:replace_right_cell)
         owner = miq_widget_set.owner
-        new_hash = {:name => "New Name", :description => "New Description", :col1 => [1], :col2 => [], :col3 => []}
-        current = {:name => "New Name", :description => "New Description", :col1 => [], :col2 => [], :col3 => []}
-        controller.instance_variable_set(:@edit, :new => new_hash, :db_id => miq_widget_set.id, :current => current)
+        new_widget = FactoryBot.create(:miq_widget)
+        new_hash = {:name => "New Name", :description => "New Description", :col1 => [new_widget.id], :col2 => [], :col3 => []}
+        controller.instance_variable_set(:@edit, :new => new_hash, :db_id => miq_widget_set.id, :current => miq_widget_set.set_data)
```

Comment:
wasn't sure if this needed a deep clone. One thing is for certain, setting this to an invalid hash (with no widgets at all) was not productive